tox.ini

```diff
@@ -2,12 +2,12 @@
 envlist =
     lint,
     py35,py36,py37,py38,pypy3,
-    docs,py36-cover,coverage,
+    docs,coverage,
 
 [testenv]
 commands =
-    cover: coverage run \
-    {envbindir}/nosetests --with-xunit --xunit-file=nosetests-{envname}.xml {posargs:}
+    python --version
+    pytest {posargs:}
```

Comment:
I don't know how much we care but those nosetests-...xml files are consumed by the pylonsproject jenkins afaik which I assume is still running. Anyway if we're getting rid of them we do need to followup and see what's going on there.

Reply:
Didn't @tseaver set that up? I couldn't find Jenkins in the repo or org config, and I can't remember where it is (was?) hosted.

Reply:
I found the URL: http://jenkins.pylonsproject.org/job/pyramid/ Linode DNS points to 45.55.247.238, which appears to be dead.

Reply:
For context: https://github.com/Pylons/pyramid/pull/2467#discussion-diff-59150297

I think it has been abandoned. I can also remove the DNS A record from Linode.

Reply:
@mmerickel I'll re-add them if there is someone that yells, I've removed them from three or four other projects now and so far I haven't heard anything.

That being said, before your comment today, the last mention of Jenkins was in 2016, and all those links are dead as Steve just mentioned.

Reply:
I deleted the A record from Linode, too. The wayback machine has only one snapshot of it from 2017.